package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "tezospayments",
+  "workspaces": [
+    "src/*"
+  ],
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.29.1",
+    "@typescript-eslint/parser": "^4.29.1",
+    "eslint": "^7.32.0",
+    "eslint-plugin-import": "^2.24.0",
+    "typescript": "^4.3.5",
+    "webpack": "4.44.2"
```

Comment:
I've added the webpack of the 4.44.2 version here for the following reason:
React scripts use this version and the landing application uses the webpack 5, thereby NPM builds the following structure:
```
tezospayments/
├── node_modules/
│   ├── react-scripts
│   └── webpack@5
└── src/
    ├── dashboard-react-web/
    │   └── node_modules/
    │       └── webpack@4
    ├── landing
    └── payment-react-web/
        └── node_modules/
            └── webpack@4
```

React scripts will try to build applications use the nearest version of the webpack, and that will be version 5. As a result, react scripts will be failed with this error: 
```
tezospayments\node_modules\react-scripts\scripts\build.js:19
  throw err;
  ^

TypeError: The 'compilation' argument must be an instance of Compilation
```

So, to avoid this error, we should say NPM to put the webpack 4 in the root node_modules, and the webpack 5 in the node_modules of the landing application:
```
tezospayments/
├── node_modules/
│   ├── react-scripts
│   └── webpack@4
└── src/
    ├── dashboard-react-web
    ├── landing/
    │   └── node_modules/
    │       └── webpack@5
    └── payment-react-web
```